src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -135,7 +128,8 @@ let require=___nfyCreateRequire(import.meta.url);
       outdir: targetDirectory,
       outExtension: { '.js': outputExtension },
       platform: 'node',
-      plugins,
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      plugins: plugins as any,
```

Comment:
Can't we just do this?

```suggestion
      plugins,
```